phpstan-baseline.neon

```diff
@@ -4,3 +4,8 @@ parameters:
 			message: "#^Class Mockery_3_App_Models_User not found\\.$#"
 			count: 1
 			path: app-modules/authorization/tests/Feature/Actions/CreatePermissionsForModelTest.php
+
+		-
+			message: "#^Undefined variable\\: \\$this$#"
+			count: 4
+			path: app-modules/knowledge-base/tests/KnowledgeBaseItem/EditKnowledgeBaseItemTest.php
```

Comment:
I believe there may be a Pest `get` method that can be imported and used in this test so we don't have to use `$this`.

If so, let's switch to that and get rid of this in the baseline. If not let me know and we can leave for now.